### DISCUSSION TITLE
Team access to Easysnmp

### DISCUSSION THREAD

**Discussion by kamakazikamikaze** [easysnmp/easysnmp#140] [General]
In the coming months, I will be moving this project under the new Easysnmp organization I've created. I hope to be able to add maintainers to teams for anyone interested that has submitted a few PRs or new features. If you'd like to request team access, please leave a comment here. (Eventually I may create an emailing list or something for people to send requests to, I dunno.)

**Comment by carlkidcrypto** [comment]
> In the coming months, I will be moving this project under the new Easysnmp organization I've created. I hope to be able to add maintainers to teams for anyone interested that has submitted a few PRs or new features. If you'd like to request team access, please leave a comment here. (Eventually I may create an emailing list or something for people to send requests to, I dunno.)

I would love to be added to the Easysnmp Org. While I haven't added any PRs or new features; I can help with testing new changes/bug fixes/ etc. Not sure if there's a tester role ?

**Reply by kamakazikamikaze** [reply]
I can probably create one so that if the role is mentioned all volunteers will receive a notification. Probably best to make one for codebase testing and another for macOS Brew testing.

**Comment by benafischer94** [comment]
@kamakazikamikaze is the intent still to move the repo over to a different organization, or keep developing here? Over the last few months I've started using the package a bit more, and would be interested in adding a hand on development / bug fixes.

**Reply by kamakazikamikaze** [reply]
The intent is to move it over. I've been juggling a lot outside of FOSS contributions so once again I let this stagnate a bit. I'll try to move some focus over to migrating the project, assuming that open issues and PRs aren't left behind when doing so

**Reply by benafischer94** [reply]
As far as I remember if you transfer the entire repo over to the new org then everything goes with it. It's been a bit since I've done this though.
I completely understand getting busy outside the FOSS things.

If there's anything I can do to assist feel free to reach out.

**Comment by kamakazikamikaze** [comment]
Repo has officially been moved to the new organization.
